app/Http/Requests/UpdateAssetRequest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Models\Asset;
+use Illuminate\Support\Facades\Gate;
+use Illuminate\Validation\Rule;
+
+class UpdateAssetRequest extends ImageUploadRequest
```

Comment:
Why are we extending ImageUploadRequest here?

Reply:
Because we still use some methods from ImageUploadRequest in the controller, this makes it so they're still available via the request - we do the same thing in `StoreAssetRequest`